.jsdoc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "tags": {
+        "allowUnknownTags": true,
+        "dictionaries": ["jsdoc"]
+    },
+    "source": {
+        "include": ["src", "package.json", "README.md"],

```

Comment:
Are `package.json` and `README.md` ignored since they don't have the `.js` pattern?

Reply:
yep!